### PR TITLE
Create index.md

### PR DESCRIPTION
**Pull request by Janeou-Zhang**
Submitting profile information for Jianou(Jane) Zhang

### CODE REVIEW

File: content/authors/Jianou Zhang/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+education: Master of Science in Statistics
+  courses:
+  - course: Computational Statistic, Bayesian Inferences, Experimental Design
+    institution: University of Saskatchewan
+
+# Social/Academic Networking, delete any items you don't want to use
+social:
+- icon: github
+  icon_pack: fab
+  link: <www.github.com/Janeou-Zhang>
```
Comment (ccontrer) [inline content/authors/Jianou Zhang/index.md:35]: maybe remove the `<` and `>`

File: content/authors/Jianou Zhang/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- name: University of Saskatchewan
+  url: http://www.usask.ca
+
+# A list of your interests, add as many as you want
+interests:
+- Bayesian Inferences
+- 
+
+# Your educational history, add as many as you want
+education: Master of Science in Statistics
```
Comment (ccontrer) [inline content/authors/Jianou Zhang/index.md:26]: perhaps this info goes into line `28`? I think there should be nothing after `education:`

**Comment by ianabc** [comment]
Yep, @ccontrer's comments are right. If you can make those edits to your file and hit commit this PR should update and we'll be able to include your changes. Nearly there!

**Comment by ianabc** [comment]
The syntax in the header region (YAML) is _very_ picky and it didn't like a couple of your edits. I've made a couple of changes and I think it's working now. We just need a picture, do you have a photo you want to use? If not, please download [this generic avatar image](https://m2pi.ca/authors/akcora/avatar.jpg) and upload that using the instructions Kristine sent. The filename should be
```
content/authors/Jianou-Zhang/avatar.jpg
```

**Comment by Janeou-Zhang** [comment]
Thank you for your work. I have added the photo in my file.

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

发件人: Ian Allison<mailto:notifications@github.com>
发送时间: 2020年8月12日 19:41
收件人: pimsmath/m2pi.ca<mailto:m2pi.ca@noreply.github.com>
抄送: Janeou-Zhang<mailto:jianouzhang2017@outlook.com>; State change<mailto:state_change@noreply.github.com>
主题: Re: [pimsmath/m2pi.ca] Create index.md (#54)


The syntax in the header region (YAML) is very picky and it didn't like a couple of your edits. I've made a couple of changes and I think it's working now. We just need a picture, do you have a photo you want to use? If not, please download this generic avatar image<https://m2pi.ca/authors/akcora/avatar.jpg> and upload that using the instructions Kristine sent. The filename should be

content/authors/Jianou-Zhang/avatar.jpg

―

**Comment by ianabc** [comment]
Looks good, thank you. I'll merge it now.

**Pull request merged**